src/ActionModal/components/ActionModal.vue

```diff
@@ -191,12 +191,12 @@
             <div slot="subtitle">
               {{ notifyMessage.body }}
               <br />
-              <br />Block
+              <br />See your transaction
```

Comment:
sorry to be annoying - but i think this whole thing should be a link "See your transaction" and we don't need the word "here".

Reply:
oh ok, makes sense. Let's change it